tests/test_pydeseq2.py

```diff
@@ -146,7 +146,7 @@ def test_deseq_independent_filtering_mean_fit(counts_df, metadata, tol=0.02):
     )
     dds.deseq2()
 
-    ds = DeseqStats(dds, contrast=["condition", "A", "B"])
+    ds = DeseqStats(dds, contrast=["condition", "B", "A"])
```

Comment:
hang on, is this what you expect? it should be [variable, baseline, group_to_compare], right? I just want to make sure I didn't mix up things in formulaic-contrasts

Reply:
It should be `[variable, group_to_compare, baseline]`, and I think that you are right and that I got confused in #328 

It used to be optional to input a contrast to `DeseqStats` before #328. I went a bit too fast in #328 and put the contrasts that passed the tests, but I realised that the code used to be 

```python
    def _contrast(self, column: str, baseline: str, group_to_compare: str) -> np.ndarray:
        """Build a simple contrast for pairwise comparisons.
        This is equivalent to
        ```
        model.cond(<column> = baseline) - model.cond(<column> = group_to_compare)
        ```
        Parameters
        ----------
        column: str
            The column to contrast.
        baseline: str
            The baseline group.
        group_to_compare: str
            The group to compare to the baseline.
        Returns
        -------
        np.ndarray
            The contrast vector.
        """
        return self.dds.cond(**{column: baseline}) - self.dds.cond(
            **{column: group_to_compare}
        )
```

Which (unless I'm mistaken again!) was wrong.

Reply:
Which is indeed what is implemented in pertpy: https://github.com/scverse/pertpy/blob/0091d0dcb6139449a5a67a1255635afd0e9e4cfb/pertpy/tools/_differential_gene_expression/_base.py#L1089